src/main/java/com/blackducksoftware/integration/hub/HubIntRestService.java

```diff
@@ -375,15 +253,15 @@ public ProjectItem getProject(final String projectUrl) throws IOException, BDRes
 
 	}
 
-	public ReleaseItem getProjectVersion(final String versionUrl)
-			throws IOException, BDRestException, URISyntaxException {
-		final ClientResource resource = createClientResource(versionUrl);
+	public ReleaseItem getProjectVersion(final String versionUrl) throws IOException, BDRestException,
+	URISyntaxException {
+		final ClientResource resource = getRestConnection().createClientResource(versionUrl);
```

Comment:
Sorry I missed that that is what I was talking about.  You already have it in place.